source/administration/object-management/object-delete.rst

```diff
@@ -0,0 +1,176 @@
+
+``DELETE`` operations that specify an object version are subject to the retention rules.
+MinIO protects object versions subject to a lock from being overwritten or deleted until the lock expires or is removed.
+
+Replicated Objects
+------------------
+
+Replication duplicates objects from one location to another.
+MinIO supports replication at the bucket level or the cluster ("site") level.
+
+Delete operations may or may not replicate, depending on the type of replication and how the replication is configured.
+
+Site Replication
+~~~~~~~~~~~~~~~~
+
+For clusters with :ref:`multi-site replication <minio-site-replication-overview>` set up, MinIO replicates all ``delete`` operations performed on any cluster to each of the other clusters in the peer group.
+
+Delete behavior on any single peer follows the same processes as any MinIO deployment.
+
+Bucket Replication
+~~~~~~~~~~~~~~~~~~
+
+With :ref:`bucket replication <minio-bucket-replication>`, MinIO supports replicating delete operations between a source bucket and a configured remote bucket.
+MinIO synchronizes deleting specific object versions *and* new  :s3-docs:`delete markers <delete-marker-replication.html>`. 
+Delete operation replication uses the same :ref:`replication process <minio-replication-process>` as all other replication operations. 
+
+MinIO requires *explicitly enabling* versioned deletes and delete marker replication. 
+Use the :mc-cmd:`mc replicate add --replicate` field to specify either ``delete`` and ``delete-marker`` or both to enable versioned deletes and delete marker replication, respectively. 
+To enable both, specify both strings using a comma separator ``delete,delete-marker``.
+
+For delete marker replication, MinIO begins the replication process after a delete operation creates the delete marker. 
+MinIO uses the ``X-Minio-Replication-DeleteMarker-Status`` metadata field for tracking  delete marker replication status. 
+In :ref:`active-active <minio-bucket-replication-serverside-twoway>` replication configurations, MinIO may produce duplicate delete markers if both clusters concurrently create a delete marker for an object *or* if one or both clusters were down before the replication event synchronized.
+
+For replicating the deletion of a specific object version, MinIO marks the object version as ``PENDING`` until replication completes. 
+Once the remote target deletes that object version, MinIO deletes the object on the source.
+While this process ensures near-synchronized version deletion, it may result in listing operations returning the object version after the initial delete operation. 
+MinIO uses the ``X-Minio-Replication-Delete-Status`` for tracking delete version replication status.
+
+MinIO only replicates explicit client-driven delete operations. 
+MinIO does *not* replicate objects deleted from the application of :ref:`lifecycle management expiration rules <minio-lifecycle-management-expiration>`. 
```

Comment:
Maybe this could be simplified, as lifecycle management isn't the immediate topic at hand.
```suggestion
MinIO does *not* replicate objects deleted by :ref:`lifecycle management expiration rules <minio-lifecycle-management-expiration>`. 
```

Reply:
Why use four words when one two-letter one does it better?